escrow/state_service_test.go

```diff
@@ -82,6 +101,18 @@ func TestGetChannelState(t *testing.T) {
 
 	assert.Nil(t, err)
 	assert.Equal(t, stateServiceTest.defaultReply, reply)
+	//Channel's nonce  =  blockchain nonce + 1
```

Comment:
I would add it as separate test case to make it possible localize error

Reply:
Done